scripts/ci-e2e.sh

```diff
@@ -70,6 +70,43 @@ source "${M3_DEV_ENV_PATH}/lib/ironic_tls_setup.sh"
 # image for live iso testing
 export LIVE_ISO_IMAGE="https://artifactory.nordix.org/artifactory/metal3/images/iso/minimal_linux_live-v2.iso"
 
+# Generate credentials
+BMO_OVERLAYS=("${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/release-0.3" "${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/release-0.4" "${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/release-0.5" "${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/release-0.6")
```

Comment:
We're not supporting 0.3 anymore, why add it?

Also, since you're using array, you can put each BMO overlay on a separate line for readability.

```suggestion
declare -a BMO_OVERLAYS=(
   aaa
   bbb
   ccc
   ...
)
```
Also, declare it as an array.